app/routes/__main/subscription/__main.tsx

```diff
@@ -0,0 +1,29 @@
+/* External dependencies */
+import React from "react";
+import {Outlet} from "@remix-run/react";
+
+/* Internal dependencies */
+import styles from "~/styles/routes/__main/profile.css";
```

Comment:
Rename the css file to `subscription.css` and add it under the `player/styles/routes/__main/subscription.css`. The content will look almost similar to that of `profile.css` in the same directory